registry/storage/driver/gcs/gcs.go

```diff
@@ -55,6 +56,7 @@ type driverParameters struct {
 	privateKey    []byte
 	client        *http.Client
 	rootDirectory string
+	projectID     string
```

Comment:
shouldn't we be using the bucket parameter instead to create the bucket name? See https://github.com/deis/distribution/pull/2/files#diff-7f96d2fa32e973ada31805a218168630R53